cartographer_ros/cartographer_ros/node.cc

```diff
@@ -131,6 +136,9 @@ Node::Node(const NodeOptions& node_options, tf2_ros::Buffer* const tf_buffer)
   wall_timers_.push_back(node_handle_.createWallTimer(
       ::ros::WallDuration(node_options_.pose_publish_period_sec),
       &Node::PublishTrajectoryStates, this));
+  wall_timers_.push_back(node_handle_.createWallTimer(
+      ::ros::WallDuration(node_options_.submap_publish_period_sec),
```

Comment:
This should get its own configuration option instead of reusing `submap_publish_period_sec`. `trajectory_publish_period_sec`?